src/biogeophys/PhotosynthesisMod.F90

```diff
@@ -586,7 +586,7 @@ subroutine InitHistory(this, bounds)
 
        endif
        this%fpsn24_patch = spval 
-       call hist_addfld1d (fname='FPSN24', units='umol CO2/m**2 ground/day',&
+       call hist_addfld1d (fname='FPSN24', units='umol CO2/m^2 ground/day',&
```

Comment:
I committed this change without recreating the .rst file and then rebuilding the documentation... I will do that next.

Reply:
Also noticing: I still need to add the code that inserts the new .rst file's title...